File: array/number_with_even_no_of_digits.py
# Given an array nums of integers, return how many of them contain an even number of digits.

def even_numbers_count(arr):
    cnt_d = 0
    for i in arr:
        cnt = 1
        while i >= 10:
            i = i//10
            cnt += 1
        if cnt % 2 == 0:
            cnt_d += 1
    return cnt_d


if __name__ == "__main__":
    arr = [123, 3456, 2343, 23]
    print(even_numbers_count(arr))

File: array/numberWithEvenNoOfDigits.js
// # Given an array nums of integers, return how many of them contain an even number of digits.

function even_numbers_count(nums) {
    let cnte=0
    for (let i =0 ; i<nums.length;i++){
        let cnt = 1
        while(nums[i] >= 10){
            Math.floor(nums[i] /= 10)
            cnt++
        }
        if (cnt % 2 == 0){cnte++}
    }
    return cnte
};

arr = [123, 3456, 2343, 23]
console.log(even_numbers_count(arr))